README.md

```diff
@@ -529,6 +529,11 @@ necessary (see [Overflow handling](#overflow-handling)). Under the hood, a chara
 is deducted column by column&mdash;the widest column being targetted each time&mdash;until
 the table will fit.
 
+To resize only specific columns, `pack` takes an `except:` arguments. E.g. `pack(except: :id)`
```

Comment:
Minor: "an `except:` arguments" should read "an `except:` argument".

Reply:
I think the README should mention the fact that `except:` can take an array.

If it's too messy to have to spell out both the array form and the symbol form, then I think it would be fine to make it so that `except:` _only_ takes an array, and not also a symbol, just for the sake of simplifying the API.

If it weren't for Rails's precedent, I would definitely prefer that this only took an array; however, so many Ruby developers are used to Rails's conventions that that constitutes a strong argument for accepting either symbol or array.

I don't have a strong preference either way&mdash;I leave it up to you.

Reply:
I've stuck with supporting both and updated the README.